magento/pwa-studio
# Null check for element styling

Fixes for https://github.com/magento/pwa-studio/issues/3013

## Description

Null check for element styling

## Related Issue

Closes #3013

## Acceptance



### Verification Stakeholders

### Specification



### Verification Steps


1. Go to a custom page built with pagebuilder with a custom CMS block
2. Verify that the page loads correctly without error
